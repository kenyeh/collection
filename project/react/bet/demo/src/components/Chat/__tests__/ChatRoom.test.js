import React from 'react';
import ChatRoom from '../ChatRoom';

describe('<ChatRoom /> should render with same capture', () => {
  it('<ChatRoom /> snapshots', () => {
    const propsData = {
      dispatch: () => {},
      messages: [],
      socket: {},
      say: () => {}
    }
    const wrapper = render(<ChatRoom {...propsData} />);
    expect(renderToJson(wrapper)).toMatchSnapshot();
  });
});