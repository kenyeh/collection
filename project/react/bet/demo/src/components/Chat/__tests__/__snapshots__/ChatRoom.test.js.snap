// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChatRoom /> should render with same capture <ChatRoom /> snapshots 1`] = `
<div
  class="chatroom"
>
  <div
    class="chatroom__chatlist"
  >
    <ul
      class="chatroom__chatlist__content"
    >
      
    </ul>
  </div>
  <div
    class="chatroom__inputbar"
  >
    <div
      class="chatroom__inputbar__form"
    >
      <input
        class="chatroom__inputbar__form__input"
        placeholder="Write a message..."
        type="text"
        value=""
      >
        
      </input>
      <div
        class="chatroom__inputbar__form__submit"
      >
        <a
          aria-disabled="false"
          class="am-button chatroom__inputbar__form__submit__btn"
          role="button"
        >
          <span>
            Send
          </span>
        </a>
      </div>
    </div>
  </div>
</div>
`;
